atomic_reactor/tasks/binary.py

```diff
@@ -37,19 +37,19 @@ class BinaryPreBuildTask(plugin_based.PluginBasedTask[PreBuildTaskParams]):
         {"name": "check_base_image"},
         {"name": "koji_parent"},
         {"name": "resolve_composes"},
-        {"name": "add_filesystem"},
         {"name": "flatpak_update_dockerfile"},
         {"name": "bump_release"},
         {"name": "add_flatpak_labels"},
         {"name": "add_labels_in_dockerfile"},
         {"name": "resolve_remote_source"},
         {"name": "pin_operator_digest"},
-        {"name": "change_from_in_dockerfile"},
         {"name": "add_help"},
         {"name": "fetch_maven_artifacts"},
         {"name": "add_image_content_manifest"},
         {"name": "add_dockerfile"},
         {"name": "inject_yum_repos"},
+        {"name": "add_filesystem"},
+        {"name": "change_from_in_dockerfile"},
         {"name": "hide_files"},
         {"name": "distribution_scope"},
         {"name": "add_buildargs_in_dockerfile"},
```

Comment:
The remaining plugins that run after `add_filesystem` are not affected by this issue?

Reply:
yes, it looks like they're not affected.